validate/validate.go

```diff
@@ -96,7 +96,11 @@ func NewValidatorFromPath(bundlePath string, hostSpecific bool, platform string)
 // CheckAll checks all parts of runtime bundle
 func (v *Validator) CheckAll() (msgs []string) {
 	msgs = append(msgs, v.CheckPlatform()...)
-	msgs = append(msgs, v.CheckRootfsPath()...)
+	if v.platform == "windows" && v.spec.Windows.HyperV != nil {
+		msgs = append(msgs, fmt.Sprintf("for Hyper-V containers, Root must not be set"))
+	} else {
+		msgs = append(msgs, v.CheckRootfs()...)
```

Comment:
You probably also want to check for an unset `root`:

```go
} else if (v.spec.Root == nil) {
  msgs = append(msgs, fmt.Sprintf("for non-Hyper-V containers, root is required"))
} else {
  msgs = append(msgs, v.CheckRootfs()...)
}
```

which will give us handling similar to that proposed for `process` in #433.